spec.bs

```diff
@@ -5030,7 +5033,7 @@ An <dfn>interest group</dfn> is a [=struct=] with the following [=struct/items=]
       [=same origin=] with [=interest group/owner=].
     </p>
   : <dfn>trusted bidding signals keys</dfn>
-  :: Null or a [=list=] of [=string=]. See [=building trusted bidding signals url=].

```

Comment:
Re-added, and fixes usages --- including the one you added, since that seems to have been confusing set to null and not set at all.  (Also changed a field nearby which was confusing not set at all to set to undefined, which are distinguishable with a bit of effort)

Reply:
We also need to update this: [0]

[0] https://github.com/WICG/turtledove/pull/1180/commits/8649f3b9b29f798873ede500dc2fbffabdbf05f1#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1492

Reply:
Not needed because allTrustedSignals being non-null implies that

Reply:
Could we add an `[=Assert=] that |ig|'s [=interest group/trusted bidding signals keys=] is non null.`?

Reply:
Ah, good idea, done